src/applications/edu-benefits/tests/wizard/containers/WizardContainer.unit.spec.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import WizardContainer from '../../../wizard/containers/WizardContainer';
```

Comment:
need to swap all of these to relative imports

Reply:
@bradl3yC Just updated.